src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -725,7 +727,7 @@ public void TriggerDatabases()
             _engine.StateMachine.DatabaseChanged += OnDatabaseChanged;
             _engine.StateMachine.ValueChanged += OnValueChanged;
 
-            _engine.TopologyChanged += OnTopologyChanged;
+            _engine.TopologyChanged += (_, __) => NotifyAboutClusterTopologyAndConnectivityChanges();
```

Comment:
```suggestion
            _engine.TopologyChanged += (_, __) => NotifyAboutClusterTopologyAndConnectivityChanges?.Invoke();
```

Reply:
`NotifyAboutClusterTopologyAndConnectivityChanges` is a method